### ISSUE TITLE
Creating a new endpoint does not add it to the routes\index.js file 

### ISSUE THREAD

**Issue by Tadimsky** [ericmdantas/generator-ng-fullstack#126]
When adding a Node.js endpoint it would be nice if the generator would automatically include the new router file in the routes\index.js file.

**Comment by Abdizriel** [comment]
@ericmdantas I can take that one after #173

**Comment by ericmdantas** [comment]
Alright.

**Comment by ericmdantas** [comment]
Out of scope.
